docs/for_users/execution_examples/run_osm.md

```diff
@@ -106,33 +106,61 @@ In your output folder `/home/geoclimate/osm/` you now have a new sub-folder call
 
 ![osm_vannes_output_files](../../resources/images/for_users/osm_vannes/osm_vannes_output_files.png)
 
-These files can be opened in most of the GIS applications (e.g QGIS, OrbisGIS, Openjump, ...).
+These files can be opened in most of the GIS applications (e.g [QGIS](https://www.qgis.org/), [OrbisGIS](http://orbisgis.org/), [OpenJUMP](http://www.openjump.org/), ...).
 
 
 
 ![osm_vannes_results](../../resources/images/for_users/osm_vannes/osm_vannes_results.png)
 
 
 
-## Using a bbox
+[Back to top](#Run-Geoclimate-with-OSM)
+
+## Using a bounding box
+
+In this example, we want to process an area (a bounding box - bbox) on top of the city center of [Paris](https://www.openstreetmap.org/relation/7444) (France). 
+
+The resulting layers will be exported as flat files in a dedicated output folder at the following address : `/home/geoclimate/osm/`
+
+### 1. Define the bbox
+
+First we have to define the desired zone and then deduce its coordinates. To do so, you can follow these steps:
+
+1. Go on the [bboxfinder.com](http://bboxfinder.com/) website,
```

Comment:
Use https://www.openstreetmap.org/export#map=10/48.0000/-1.0959
Select the zoon and clic on export you will have the osm bbox

Reply:
If you click on export, you ask to download the data (and I get an error, since there is too many objects).
It's the same approach. The screenshots are done. No time to change right now